### ISSUE TITLE
Diminish focus on X-risks on the website

### ISSUE THREAD

**Issue by Pato-desu** [PauseAI/pauseai-website#242]
This is something that I don't think is a priority, and that I have also contributed to, but the PauseAI website is super X-risks coded.
I think we could get more buy-in if we just added more content about present harms, but more easily and importantly, if we just mentioned "Dangers of AI" instead of using catastrophic expressions here and there.
Examples:

- [ ] Frontpage
- [ ] /about
- [ ] /quotes
- [ ] /learn
- [ ] /environmental
- [ ] Adding pages on the present harms being done

**Comment by student-sx** [comment]
I think the x-risk is the most important point to address. That doesn't mean that other risks aren't important. The industry is aiming to build AGI and ASI with great effort. If this aim will be reached we have a significant x-risk.

**Comment by Pato-desu** [comment]
Yeah, but I mean, look at https://pauseai.info/about for example. Completely not inclusive of other worries

**Comment by anthonybailey** [comment]
We are being extinction risk coded because the greatest driver and most convincing argument for any international moratorium on racing to superintelligence is that everybody dying would be really terrible.

If you are saying we should remind folk that regulation and slowdown can help when developing mitigations for other large risks too, we agree. If we're missing clear opportunities to do so, we can fix them.

The thing that would worry me is if we appear to be an umbrella for all AI risks in a way that leads to extinction not being a normal thing to shout about.

If anyone wants to campaign for a moratorium only for other reasons, we will happily share protest opportunities with them, discuss how aims overlap, and express supportive opinions on risks most agree are serious.

But there is harmful constant pressure to, shall we say, "water down" loss of control risks for sake of engagement. We have to recognize that by default we are harmed by that and have to fight it.

Messaging in ways that recognize that risks of death provoke copium and resistance and attempt to help folk push through that is grand. It's my passion project.

But if our movement ends up in any sense trying not to talk about extinction we've blown it.

**Comment by Pato-desu** [comment]
more evidence in my direction https://youtu.be/pquU5SdspEg?si=aUK-Eh2M3fcs7hqc&t=2970

**Comment by Pato-desu** [comment]
it feels like rationalists care more about being right or "truthful" than not dying
